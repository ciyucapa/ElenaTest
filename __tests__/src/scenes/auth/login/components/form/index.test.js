import React from 'react';
import {create} from 'react-test-renderer';

import Form from '../../../../../../../src/scenes/auth/login/components/form';

describe('Login form component', () => {
    test('render login form component', () => {
        const tree = create(
            <Form
                isValidForm
                phone="+5748545852"
                password="123456789"
            />,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});