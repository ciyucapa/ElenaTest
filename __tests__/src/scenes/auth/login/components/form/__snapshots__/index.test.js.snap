// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login form component render login form component 1`] = `
<div
  className="sc-bdfBQB lgbBAZ"
>
  <div
    className="sc-gsTEea ipUvat"
  >
    <div
      className="sc-dlfnuX khnmHO"
    >
      <input
        onChange={[Function]}
        placeholder="Telefono"
        type="text"
        value="+5748545852"
      />
    </div>
    <div
      className="sc-dlfnuX khnmHO"
    >
      <input
        onChange={[Function]}
        placeholder="Contraseña"
        type="password"
        value="123456789"
      />
    </div>
    <div
      className="sc-hKgJUU fXEUha"
    >
      <button
        className="sc-iBPTik xaJDg"
        disabled={false}
        onClick={[Function]}
      >
        Iniciar Sesión
      </button>
    </div>
  </div>
</div>
`;
